articles/Silo_Finance.md

```diff
@@ -0,0 +1,388 @@
+
+
+
+![Silo_Repository_immutable](https://user-images.githubusercontent.com/89845409/200543353-4a07e9fc-c0ac-4e5f-bb65-2a6319024608.png)
+
+(source: [SiloLens.sol](https://etherscan.io/address/0xf12C3758c1eC393704f0Db8537ef7F57368D92Ea#code))
+
+
+
+* [Silo](https://etherscan.io/address/0xd998C35B7900b344bbBe6555cc11576942Cf309d#code) is the main component of the protocol that acts as a vault for assets, implementing the lending logic, managing and isolating the risk, and performing liquidations.
+* [PriceProviderRepository](https://etherscan.io/address/0x7C2ca9D502f2409BeceAfa68E97a176Ff805029F#code) manages the oracle modules and the price request routing for each silo. It can support many protocols and sources. Currently, it supports three oracle sources: [Chainlink](https://etherscan.io/address/0xe37B8c83138caF12E57632D19c06Eb561D47e423#code) (default price provider), UniswapV3 TWAP, and BalancerV2 TWAP.
+* [Interest Rate Model](https://etherscan.io/address/0x7e9e7ea94e1ff36e216a703d6d66ece356a5fd44#code) Silo Finance uses a dynamic interest rates model which is described in more detail in this [Interest Rate Model paper](https://drive.google.com/file/d/11jzbdIQ9KGm_ZVFIUwUu44O1ktWx5R30/view?pli=1).
+* [Silo Router](https://etherscan.io/address/0xb2374f84b3cEeFF6492943Df613C9BcF45322a0c#code) is a utility contract that can batch any number or combination of actions (Deposit, Withdraw, Borrow, Repay) and execute in a single transaction.
+* The permission system registry consists of three smart contracts: GuardedLaunch.sol, TwoStepOwnable.sol, and Manageable.sol
+
+
+![Silo_protocol_architecture](https://user-images.githubusercontent.com/89845409/200553725-5b4d8b26-ee1b-4914-b6c2-a1ec803063a0.png)
+
+(source: [Certora Formal Verification Document](https://www.silo.finance/_files/ugd/fd5034_98066b4c70ea46b5862a00670179f067.pdf))
+
+
+# SILO Governance
+
+The governance process follows a standard procedure of forum discussion, Snapshot vote, and finally on-chain vote. For more details visit the [docs](https://silopedia.silo.finance/governance/silodao). In practice, however, Snapshot is mostly applied for decisions concerning expenditures of the DAO’s funds, whereas changes that affect the protocol itself are directly voted on via an on-chain proposal on [Tally](https://www.tally.xyz/governance/eip155:1:0xA89163F7B2D68A8fbA6Ca36BEEd32Bd4f3EeAf61). The funds are managed by a 2-of-3 multi-sig (owners: [signer1](https://etherscan.io/address/0x9b8b04B6f82cD5e1dae58cA3614d445F93DeFc5c), [signer2](https://etherscan.io/address/0x66B416a3114A737f0353DC74d1E12a7e23f686F9), [signer3](https://etherscan.io/address/0xe153437bC974cfE3E06C21c08AeBbf30abaefa2E)).
+
```

Comment:
If possible, also link the names / twitter of the msig signers (No need to link address to name, just the names enough)

Same for sources @ very bottom of the report

Reply:
Let's see if Aiham comes back on this, will add once he does.